include/votca/tools/application.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009-2011 The VOTCA Development Team (http://www.votca.org)
+ * Copyright 2009-2018 The VOTCA Development Team (http://www.votca.org)
```

Comment:
Huh, why the update? You didn't touch that file!

Reply:
Just leave it, it won't hurt. Next time use something like:
```
git diff --name-only origin/master | xargs sed -i '/Copyright/s/-20../-2018/'
```

Reply:
Is there anyway we can automate this, whenever a push is made. So the files that were changed reflect the dates correctly?

Reply:
You could make the above command a git pre-commit hook.